### DISCUSSION TITLE
PhotoPrism optional config does not pull any data

### DISCUSSION THREAD

**Discussion by kmanan** [linuxserver/Heimdall#1154] [General]
Test says Successfully communicated with the API but the photos and videos counter is 0

**Comment by TK-3375** [comment]
Just to bring more attention to that one : I have the same issue here. Installed through docker (both), no stats

**Comment by svanhoutte** [comment]
check this https://github.com/linuxserver/Heimdall-Apps/issues/496 that will solve the issue
